src/com/phronemophobic/membrane/term.clj

```diff
@@ -317,47 +320,59 @@
     (color-scheme/load-scheme source)
     default-color-scheme))
 
+(defn font-exists? [font-family font-size]
+  (or (= "monospace" font-family)
```

Comment:
It seems like this check is only for the `validate-font` function use case. If that's the case, I think we should move the "monospace" check to `validate-font`. If that's not the case, then I think we should give use a different name here besides `font-exists?` since there is another function named `font-exists?` that does something very slightly different.

Reply:
Hmm... yeah... 

I think I was imagining someone coming in direct to the Clojure API instead of via the command line. 

So... 
- coming in from command-line, if the requested font is not available we return a nice error.
- but coming in from the API, if the requested font is not available we default to "monospace" 12. 

I cannot remember why I thought defaulting was a good idea. 
I might have been trying to match pre-existing behavior-ish.
Personally, if I've requested a font that is not available, I'd rather that things fail than things being "helpful".

### Proposed change 1
So, because:

```Clojure
(skia/font-exists? (ui/font "monospace" 12))
;; => false
```
(I think `(ui/font "monospace" 12)` is valid yeah? Will choose the system default monospace font?)

I think we still need the equivalent of `...membrane.term/font-exists?`.  
How about a rename to `font-available?`.
`font-valid?` would also work.

### Proposed change 2
I think that `load-terminal-font` should throw if the requested font is not available.

Watcha think?

Reply:
This is an issue I ran into with skia as well. If you try to load a font by font family, it will "do its best" and always return something. Currently, `font-exists?` just checks whether font family of the loaded font matches the font family provided.

I wonder if there's a way to query the system for monospaced fonts in which case we could use the default actual mono spaced font family as the font rather than "monospace". Let me check!

Reply:
Interesting:
https://github.com/google/skia/blob/main/src/ports/SkFontConfigInterface_direct.cpp#L214

Reply:
This seems to work:
```
> (skia-font-family-name
                (get-font (ui/font "monospace" 12)))
"Courier"
```
Maybe it makes sense for skia to offer the following public functions that return the font family
- `default-font-family` : returns the default system font with font family included
- `default-serif-font-family`
- `default-sans-serif-font-family`
- `default-monospace-font-family`

With `default-monospace-font-family`, we can use that as the default font family rather than monospaced.

Thoughts?

Reply:
I think that works, yeah.

I like that we can get at the actual font family that the font class `monospace` represents. 

I wonder if a single `default-font-family` fn with a `font-class` arg might be slightly more usable/extensible?

Reply:
> I wonder if a single default-font-family fn with a font-class arg might be slightly more usable/extensible?

Yea, I like idea. I'll go with that. I'll try to release that update in membrane soon!

Reply:
Cool, in the meantime, I'll carry out the 2 proposed changes in an upcoming commit to this PR. When you release a membrane update we can circle back and tweak.

Reply:
Done